### ISSUE TITLE
SliverCrossAxisConstrained but align left?

### ISSUE THREAD

**Issue by ChauCM** [Kavantix/sliver_tools#43]
This is what I try to do:
![ezgif-2-121711b582](https://user-images.githubusercontent.com/86839861/155660526-175a8143-7767-4565-b458-c6cfc7caf555.gif)
```
class WidgetThatReturnsASliver extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return SliverStack(
      insetOnOverlap: false, // defaults to false
      children: <Widget>[
        SliverPositioned(
           top: 0,
            right: 0,
            width: 300,
          child: Container(
            width: 300,
            height: 600,
            decoration: BoxDecoration(
              color: Colors.red,
              ],
              borderRadius: BorderRadius.circular(8),
            ),
          ),
        )

        SliverList(...), // I want this SliverList stay left, with a width of 700
      ],
    );
  }
}
```
I try wrap SliverList With A SliverPositioned(top: 0, left: 0, width 700), my sliverList take up the whole screen
I try SliverCrossAxisConstrained(max: 700) then my sliverList stay in the middle. 
How can I achieve the UI?

**Comment by Kavantix** [comment]
You can try using `SliverCrossAxisPadded` and give it a `paddingEnd` of the size of the box you want on the right.

**Comment by Kavantix** [comment]
@ChauCM can you test with PR #47

**Comment by Kavantix** [comment]
@ChauCM this feature is now included in v0.2.6
